nixos/modules/config/system-path.nix

```diff
@@ -41,12 +41,17 @@ let
       pkgs.zstd
     ];
 
-    defaultPackages = map (pkg: setPrio ((pkg.meta.priority or 5) + 3) pkg)
-      [ pkgs.nano
-        pkgs.perl
-        pkgs.rsync
-        pkgs.strace
-      ];
+  defaultPackageNames =
+    [ "nano"
+      "perl"
+      "rsync"
+      "strace"
+    ];
+  defaultPackages =
+    map
+      (n: let pkg = pkgs.${n}; in setPrio ((pkg.meta.priority or 5) + 3) pkg)
+      defaultPackageNames;
+  defaultPackagesText = "[ ${concatMapStringsSep " " (n: "pkgs.${n}") defaultPackageNames } ]";
```

Comment:
Not mentioning the priority change possibly confusing?

Reply:
could be, however it also isn't mentioned currently. how turning the text into a docbook literal:
```
these packages, with their <literal>meta.priority</literal> increased:
<programlisting>${defaultPackagesText}</programlisting>
```

Reply:
Sounds good to me, though increasing `meta.priority` actually [lowers the priority](https://nixos.org/manual/nix/stable/command-ref/nix-env.html?highlight=priority#description-1) so it might be confusing.

Reply:
curse priorities using numerically increasing sort order :/

another suggestion:
```
these packages, with their <literal>meta.priority</literal> numerically increased (thus lowering their installation priority):
<programlisting>${defaultPackagesText}</programlisting>
```